uint8 leg_type
float32[3] q
float32[3] dq
float32[3] tau